tests/hello-example.js

```diff
@@ -1,71 +1,188 @@
 const { test } = require("ava");
-const { instantiate } = require("../src/wasm");
+const loadRust = require("../src/compile");
 const wrap = require("../src/wrap");
 
-test.beforeEach(async (t) => {
-  t.context.i = await instantiate("./hello-wasm.wasm");
-});
+const allocFunction = `
```

Comment:
👍

Reply:
Feel free to add use `std::mem; use std::os::raw::{c_char, c_void};`  as well

Reply:
yeah could have done that 👍

Reply:
`std::meme`